include/oneapi/dpl/internal/exclusive_scan_by_segment_impl.h

```diff
@@ -65,7 +65,11 @@ exclusive_scan_by_segment_impl(Policy&& policy, InputIterator1 first1, InputIter
     flags[0] = 1;
 
     transform(::std::forward<Policy>(policy), first1, last1 - 1, first1 + 1, _flags.get() + 1,
-              ::std::not2(binary_pred));
+              [binary_pred](
```

Comment:
Actually you don't need to introduce new lambda-wrapper here.
There is `class __not_pred` defined in the file oneDPL\include\oneapi\dpl\pstl\utils.h

Reply:
Applied.